src/metabase/util/string.clj

```diff
@@ -29,3 +30,19 @@
        (= (count parts) 1) (str (first parts) (when stop? \.))
        (= (count parts) 2) (str (first parts) " " (deferred-tru "and")  " " (second parts) (when stop? \.))
        :else               (str (first parts) ", " (build-sentence (rest parts) options))))))
+
+
+(defn mask
```

Comment:
nice, looks like slack-app-token use this feature as well, maybe we could re-use this helper fn there ? https://github.com/metabase/metabase/blob/d8322e9a89b65835d4c3726e7dd4131ca8c7d744/src/metabase/integrations/slack.clj#L33

out of scope of this PR so up to you if you want to do it.